docs/Store.md

```diff
@@ -53,16 +55,48 @@ render items using this mapping.
 Once created the render items array is passed into the `initItems()` function and by default store in the
 `renderItems` property.
 
-In addition to this, if the store is an observable store (`dstore/Trackable`) the changes to the data in the store will
-be tracked and the following functions will be called on each type of modification:
+## Observability/Trackability
+
+In addition to this, the source can be observable.
+There are various possibilities to make the source observable.
+
+#### Observable source with `dstore/Trackable` in source property
+
+The first one is to use a `dstore/Trackable` in the source.
+By using that, the changes to the data in the source will be tracked and the following functions will be called on each
+type of modification:
 
 * `itemRemoved` if an item has been removed
 * `itemAdded` if an item has been added
 * `itemUpdated` if an item has been updated (its properties have changed)
-* `itemMoved` if an item has been moved in an ordered store.
+* `itemMoved` if an item has been moved in an ordered store
 
 By default those functions update the `renderItems` accordingly.
 
+#### Observable source with an array in source property
```

Comment:
If I'm not mistaken, what you are asking here is some enhancement of this documentation that would also have applied to the previous version (i.e. with dstore). In other words that is not specific to @harbalk work. Also knowing that @harbalk only have limited time until the end of his internship now (basically one week), I would suggest we skip this for now and we open a separate issue.